src/app/pages/courses/pipes/orderby.pipe.ts

```diff
@@ -0,0 +1,22 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { Course } from 'app/shared/models/course.model';
+
+@Pipe({
+    name: 'orderByDate'
+})
+
+export class OrderByPipe implements PipeTransform {
+    public transform(courses: Course[], direction: string) {
+        if (direction === 'asc') {
+            return courses.sort((a: Course, b: Course) => {
+                return a.createdDate.getTime() - b.createdDate.getTime();
```

Comment:
The pipe is named `orderBy` meaning that it should accept a parameter by which the ordering happens. You have hardcoded the `createdDate` here, coupling the pipe with your business domain.